Website/src/rendering/src/assets/css/components/PageContent/TopBanner.css

```diff
@@ -0,0 +1,15 @@
+.bx_TopBanner {
```

Comment:
Defining Boxever classes in our bundle will cause issues when we use Boxever web experiences. These web experience will then use our styles plus the ones defined in the experience.

Please rename all of these classes according to our standards: e.g. dynamic-welcome-message, message-banner